src/params/checkoutSessionId.js

```diff
@@ -1,4 +1,4 @@
 /** @type {import("@sveltejs/kit").ParamMatcher} */
 export function match(param) {
-  return /^cs_[a-zA-Z0-9_-]+$/.test(param);
+  return /^(?:cs_)?[a-zA-Z0-9_-]+$/.test(param);
```

Comment:
since this appears to be a regex matcher, it probably makes sense to length limit here to 36 characters to avoid a ReDoS issue